src/main/java/core/basesyntax/ShopServiceImpl.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class ShopServiceImpl implements ShopService {
```

Comment:
The `ShopServiceImpl` class should not be responsible for holding the storage state. It would be better to create a separate class, for example, `Storage` in a `db` package to encapsulate the storage logic and make `ShopServiceImpl` independent of it, following the SOLID principles.